### PR TITLE
issue #78  Added SVN 

### PR DESCRIPTION
**Pull request by thisisabdus**
NOTE:
- Please fill out the template below 

## Did you read the instructions in the [README](https://github.com/the-ethan-hunt/first-timers-guide/blob/master/README.md)?

> Yes

## Have you read the instructions in [CONTRIBUTING](https://github.com/the-ethan-hunt/first-timers-guide/blob/master/CONTRIBUTING.md)?

> Yes

## Have you written this article by yourself? Have you taken inspiration from anywhere and then written it in your words?

> I have written from my own experience

### CODE REVIEW

**Comment by thisisabdus** [comment]
@the-ethan-hunt, let me know if any modifications needed.

File: guides/Version_Control/svn.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+| `git clone <repo>`          | `svn co <repo>` |
+| `git add .`                 | `svn add *` _(wildcard for adding all un-added files)_
+| `git commit -m "Message"`   | `svn commit -m "Message"`
+| `git push`                  | `svn update`
+| `git pull`                  | `svn update`
+| `git status`                | `svn status`  
+| `git log`                   | `svn log`
+
+___
+###### written by [Abdus](https://abdusdev.me)
```
Comment (thisisabdus) [inline guides/Version_Control/svn.md]: Let me know if it isn't allowed. I will remove it.

Comment (the-ethan-hunt) [inline guides/Version_Control/svn.md]: Remove this line

Comment (thisisabdus) [reply guides/Version_Control/svn.md]: Sure.

**Comment by thisisabdus** [comment]
Done @the-ethan-hunt

**Pull request merged**